Sources/Geocodable.swift

```diff
@@ -22,52 +22,81 @@ public protocol Geocodable {
     func geocodeLocality(_ string: String,
                          forCountry country: String,
                          completion: @escaping (Result<GeocodedAddress, GeocodingError>) -> ())
-
-}
-
-public extension Geocodable {
-
-    /// Geocodes the given string into a geocoded address.
+    
+    /// Geocodes the given string into a geocoded location.
     ///
     /// - Parameters:
     ///   - string: The string to geocode.
     ///   - country: The country to use as a filter. Default value is "US".
     ///   - completion: The completion block.
     func geocodeLocality(_ string: String,
```

Comment:
Since we are still in alpha version, it's expected to bring breaking changes to the code. Also I'd like to limit the amount of public APIs available to simplify the code and learning curve for the users. Can you merge your new functionality with the existing one by using 1 public function and update the existing model?